man/en/sos.conf.5

```diff
@@ -51,19 +119,23 @@ threads=10
 .sp
 To disable the 'host' and 'filesys' plugins:
 .LP
-[plugins]
+[report]
 .br
-disable = host, filesys
+noplugins = host,filesys
```

Comment:
This is renamed to `skip-plugins` else-where.

I see this renaming bit controversial, as 1) it is backward incompatible, 2) short option `-n` is not a shortcut from `--skip-plugins` (while we already use `-s` for `--sysroot`). So (not ultimate) +1 for keeping the old "noplugins" or could-be "no-plugins" option.

Reply:
Hmm. I don't think we'll be able to avoid point #1, but as this is a major version update I'm also not too concerned about that. 

The problem I see is that both the short- and long-form name make sense in their own context. `-n` as a "do *n*ot use this" is fairly conventional. `skip-plugins` is also straight forward and intuitive.

However, a long form of `no-plugins` doesn't sound right to an English speaker. I wouldn't say "run sos with no plugins kvm, host, logs", and the first thing that comes to mind with `no-plugins` is "oh, this option means disable all plugins" which obviously isn't the case.